### ISSUE TITLE
COT type t-x-d-d  message no handled.

### ISSUE THREAD

**Issue by erichexter** [darkplusplus/cot#2]
It looks like when ATAK and iTak clients disconnect from the server, this message with the link element is sent out. The clients use this to deactivate a contact marker. 

`<event how="h-g-i-g-o" type="t-x-d-d" version="2.0" uid="d57b840d-b971-4cf5-9aac-377cfcf844e8" start="2023-08-18T02:03:18Z" time="2023-08-18T02:03:18Z" stale="2023-08-18T02:03:38Z">
<point ce="9999999" le="9999999" hae="0" lat="0" lon="0" />
<detail>
    <link relation="p-p" uid="ABAD7892-3FD6-4F45-A57B-AF3DF0286DE2" type="a-f-G-E-V-C" />
    <_flow-tags_ TAK-Server-f6edbf55ccfa4af1b4cfa2d7f177ea67="2023-08-18T02:03:18Z" />
</detail>
</event>`

Just wanted to document it as a known issue. I am not sure how the protobuf definitions were created but I changed my usage to parse this type separately.

**Comment by gregbuehler** [comment]
tl;dr - the predicates are [code generated](https://github.com/darkplusplus/cot/blob/main/dpp.cot/CotPredicates.tt) from a master document.

My understanding is that the `t-x-` prefix roughly maps to "tasking-experimental". From what you're saying this should likely map to a client disconnect.
